examples/tutorials/mnist_pytorch/adaptive.yaml

```diff
@@ -29,5 +26,5 @@ searcher:
   smaller_is_better: true
   max_trials: 16
   max_length:
-    batches: 937 #60,000 training images with batch size 64
```

Comment:
pretty sure that asha + max length of 1 is going to behave poorly.  The master doesn't do fractions, so asha will just emit everything with epoch length of 1.

Use batches here instead, I think.